### PR TITLE
[Emoji] Update slack data

### PR DESCRIPTION
**Pull request by alamirault**
| Q             | A
| ------------- | ---
| Branch?       | 7.1
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | -
| License       | MIT

Update slack data to latest with make build

Split https://github.com/symfony/symfony/pull/54441 in 2 PR as suggested in https://github.com/symfony/symfony/pull/54441#discussion_r1546252696

### CODE REVIEW

File: src/Symfony/Component/Emoji/CHANGELOG.md [hunk 1]
```diff
@@ -5,3 +5,4 @@ CHANGELOG
 ---
 
  * Add the component
+ * Update `slack` data
```
Comment (derrabus) [inline src/Symfony/Component/Emoji/CHANGELOG.md]: This entry is a bit pointless given that we haven't shipped a stable version of the component yet that would've included the old data set.

**Review by derrabus — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Emoji/CHANGELOG.md (hunk 1 above)
Comment (alamirault) [reply src/Symfony/Component/Emoji/CHANGELOG.md]: Thank you for explanation, updated

**Review by javiereguiluz — APPROVED** [review]

**Review by lyrixx — APPROVED** [review]
Thanks

**Review by nicolas-grekas — COMMENTED** [review]
For 6.4 ?

**Comment by stof** [comment]
@nicolas-grekas as the data has been extracted in a standalone component in 7.1, I think it is easier to keep this update in 7.1.

**Comment by nicolas-grekas** [comment]
Thank you @alamirault.

**Pull request merged**